es/13_0_Disenando_Bitcoin_Scripts_Reales.md

```diff
@@ -0,0 +1,21 @@
+# Capítulo 13: Diseño de scripts de Bitcoin reales
+
+Nuestros scripts de Bitcoin hasta la fecha han sido en gran parte ejemplos teóricos, porque todavía hemos estado juntando las piezas del rompecabezas. Ahora, con el repertorio completo de Bitcoin Script en la mano, estamos listos para profundizar en varios Scripts de Bitcoin del mundo real y ver cómo funcionan.
```

Comment:
`Ahora, con el repertorio completo de Bitcoin Script en la mano, estamos listos para profundizar en varios scripts de Bitcoin del mundo real y ver cómo funcionan.`

Reply:
This change is about the capital letter at the beginning of the word `scripts` in `...varios Scripts de Bitcoin...` ( I think that it should be changed to `...varios scripts de Bitcoin...`) but if you think that it shouldn't be changed it's okay.